themes/custom/az_barrio/css/style.css

```diff
@@ -641,7 +645,37 @@ fieldset.collapsed {
   width: initial;
   max-width: 100%;
 }
+}
```

Comment:
Should the media query closing bracket be this far down?

Reply:
honestly, I don't really know where it needs to be, there wasn't one and so everything above it was already being set on that media query, and I figured  adding it there would do the least amount of harm